tools/ci_build/amd_hipify.py

```diff
@@ -182,6 +182,8 @@ def hipify(hipify_perl_path, src_file_path, dst_file_path):
     s = s.replace("rocm_device_prop_", "cuda_device_prop_")
     s = s.replace("rocm_device_arch_", "cuda_device_arch_")
 
+    s = s.replace("HipTuningContext", "RocmTuningContext")
```

Comment:
Why is this need? If it is a problem, we should have already been experiencing CI failures.

Reply:
`HipTuningContext` is hipified from `CudaTuningContext `, but `RocmTuningContext` is what we need. We didn't encounter this problem before because rocm TunableOp didn't reuse cuda code.